### ISSUE TITLE
uninitialized constant Sidekiq::Component after upgrading from 6.5.1 to 6.5.4

### ISSUE THREAD

**Issue by denisj** [mperham/sidekiq#5462]
Ruby version: 3.1.2
Rails version: 7.0.3.1
Sidekiq / Pro / Enterprise version(s): 6.5.4 (normal version)
OS : Ubuntu 18.04.6 LTS

After upgrading to 6.5.4 and running `bundle exec rails test`, I got this error:

### Backtrace
```
/home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/sidekiq-6.5.4/lib/sidekiq/job_retry.rb:66:in `<class:JobRetry>': uninitialized constant Sidekiq::Component (NameError)

    include Sidekiq::Component
                   ^^^^^^^^^^^
Did you mean?  Complex
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/sidekiq-6.5.4/lib/sidekiq/job_retry.rb:61:in `<module:Sidekiq>'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/sidekiq-6.5.4/lib/sidekiq/job_retry.rb:6:in `<main>'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/sidekiq-status-2.1.3/lib/sidekiq-status/server_middleware.rb:2:in `<main>'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/sidekiq-status-2.1.3/lib/sidekiq-status.rb:6:in `<main>'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bundler-2.3.11/lib/bundler/runtime.rb:60:in `block (2 levels) in require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bundler-2.3.11/lib/bundler/runtime.rb:55:in `each'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bundler-2.3.11/lib/bundler/runtime.rb:55:in `block in require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bundler-2.3.11/lib/bundler/runtime.rb:44:in `each'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bundler-2.3.11/lib/bundler/runtime.rb:44:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bundler-2.3.11/lib/bundler.rb:176:in `require'
	from /home/djean/workspace/conversation-stable/config/application.rb:23:in `<main>'
	from /home/djean/workspace/conversation-stable/config/environment.rb:2:in `require_relative'
	from /home/djean/workspace/conversation-stable/config/environment.rb:2:in `<main>'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/workspace/conversation-stable/test/test_helper.rb:22:in `<main>'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/workspace/conversation-stable/test/controllers/admin/networks_controller_test.rb:1:in `<main>'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/railties-7.0.3.1/lib/rails/test_unit/runner.rb:47:in `block in load_tests'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/railties-7.0.3.1/lib/rails/test_unit/runner.rb:47:in `each'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/railties-7.0.3.1/lib/rails/test_unit/runner.rb:47:in `load_tests'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/railties-7.0.3.1/lib/rails/test_unit/runner.rb:40:in `run'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/railties-7.0.3.1/lib/rails/commands/test/test_command.rb:33:in `perform'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/railties-7.0.3.1/lib/rails/command/base.rb:87:in `perform'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/railties-7.0.3.1/lib/rails/command.rb:48:in `invoke'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/railties-7.0.3.1/lib/rails/commands.rb:18:in `<main>'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from /home/djean/software/ecairn/rvm/gems/ruby-3.1.2@rails7.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
	from script/rails:8:in `<main>'
```

### sidekiq.yml
```
development:
  :require: /home/djean/workspace/conversation-stable/config/environment.rb
  :client_pool_size: 1
  :server_pool_size: 13
  :timeout: 10
  :concurrency: 10

test:
  :require: /home/djean/workspace/conversation-stable/config/environment.rb
  :client_pool_size: 1
  :server_pool_size: 13
  :timeout: 10
  :concurrency: 10

production:
  :require: /home/djean/workspace/conversation-stable/config/environment.rb
  :client_pool_size: 1
  :server_pool_size: 13
  :timeout: 10
  :concurrency: 10

:queues:
  - mailer
  - [network_report, 2]
  - [digest, 2]
  - [top_articles_report, 4]
  - [enrichment, 4]
  - [labeling, 4]
  - [classification, 4]
  - [twitter_api, 4]
  - [import, 4]
  - [prediction, 4]
  - [adv_app, 4]
  - [affluent_kb, 4]
  - [expand, 5]
  - [compute_metrics, 5]
  - [compute_relationships, 5]
  - [other, 5]

:limits:
  network_report: 2
  digest: 2
  twitter_api: 2
  labeling: 2
  classification: 2
  enrichment: 2
  import: 2
  prediction: 2
  affluent_kb: 2

:process_limits:
  network_report: 1
  digest: 1
```

### Initializer
```
require 'fileutils'
require 'redis/connection_pool'
require 'sidekiq/admin'
require 'sidekiq/rails'
require 'sidekiq-status'
require 'sidekiq/middleware/server/expiring_jobs'
require 'sidekiq/middleware/server/sentry_error_logger'
require 'sidekiq/extensions/action_mailer'

# exception_notification provides a Sidekiq middleware (server)
# that notify in case of exception by passing Sidekiq data.
require 'exception_notification/sidekiq'

Redis.exists_returns_integer = true

# see https://github.com/mperham/sidekiq/blob/v3.3.2/lib/sidekiq/rails.rb#L35
# see https://github.com/mperham/sidekiq/wiki/Delayed-extensions
# Delayed extensions are disabled by default in sidekiq5+
# Sidekiq.remove_delay!

ActiveSupport.on_load(:action_mailer) do
  extend Sidekiq::Extensions::ActionMailer
end

# https://github.com/mhenrixon/sidekiq-unique-jobs/blob/master/README.md#add-the-middleware
Sidekiq.configure_server do |config|
  # re-define Rails.logger as Sidekiq logger
  ::Rails.logger = Sidekiq.logger

  config.redis = Redis.connection('sidekiq', Sidekiq::Admin.server_pool_size)

  config.client_middleware do |chain|
    chain.add SidekiqUniqueJobs::Middleware::Client
    chain.add Sidekiq::Status::ClientMiddleware, expiration: 30.minutes
  end

  config.server_middleware do |chain|
    chain.add Sidekiq::Middleware::Server::ExpiringJobs
    chain.add Sidekiq::Middleware::Server::SentryErrorLogger
    chain.add Sidekiq::Status::ServerMiddleware, expiration: 30.minutes
    chain.add SidekiqUniqueJobs::Middleware::Server

    if Sidekiq::Admin.profile_every
      require 'sidekiq/middleware/server/memory_profiler'
      heap_dir = File.join(::Rails.root, 'tmp', 'sidekiq', ::Rails.env)
      ::FileUtils.mkdir_p heap_dir

      chain.add Sidekiq::Middleware::Server::MemoryProfiler,
        every: Sidekiq::Admin.profile_every,
        dir: heap_dir

      Sidekiq.logger.info "Memory profiler configured to dump heap every #{Sidekiq::Admin.profile_every} job(s)"
    end
  end

  SidekiqUniqueJobs::Server.configure(config)

  # Change DB pool size for ActiveRecord
  # https://github.com/mperham/sidekiq/issues/503
  if defined?(ActiveRecord::Base)
    ActiveRecord::Base.connection_pool.disconnect!
    db_config = ::Rails.application.config.database_configuration[::Rails.env]
    db_config['pool'] = Sidekiq::Admin.concurrency + 2 # Give it some buffer
    ActiveRecord::Base.establish_connection(db_config)
  end
end

Sidekiq.configure_client do |config|
  config.redis = Redis.connection('sidekiq', Sidekiq::Admin.client_pool_size)

  config.client_middleware do |chain|
    chain.add SidekiqUniqueJobs::Middleware::Client
    chain.add Sidekiq::Status::ClientMiddleware, expiration: 30.minutes
  end
end

# change default sidekiq options for Sidekiq::Extensions::DelayedMailer
module Sidekiq
  module Extensions
    class DelayedMailer
      sidekiq_options queue: :mailer, retry: true
    end
  end
end
```

**Comment by krystof-k** [comment]
I bumped into the same issue when running `assets:precompile`:

```
#0 4.128 NameError: uninitialized constant Sidekiq::Component
#0 4.128 Did you mean?  CompositeIO
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/sidekiq-6.5.4/lib/sidekiq/job_retry.rb:66:in `<class:JobRetry>'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/sidekiq-6.5.4/lib/sidekiq/job_retry.rb:61:in `<module:Sidekiq>'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/sidekiq-6.5.4/lib/sidekiq/job_retry.rb:6:in `<main>'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/sidekiq-status-2.1.3/lib/sidekiq-status/server_middleware.rb:2:in `<main>'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/sidekiq-status-2.1.3/lib/sidekiq-status.rb:6:in `<main>'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
#0 4.128 /usr/local/bundle/gems/bundler-2.2.31/lib/bundler/runtime.rb:60:in `block (2 levels) in require'
#0 4.128 /usr/local/bundle/gems/bundler-2.2.31/lib/bundler/runtime.rb:55:in `each'
#0 4.128 /usr/local/bundle/gems/bundler-2.2.31/lib/bundler/runtime.rb:55:in `block in require'
#0 4.128 /usr/local/bundle/gems/bundler-2.2.31/lib/bundler/runtime.rb:44:in `each'
#0 4.128 /usr/local/bundle/gems/bundler-2.2.31/lib/bundler/runtime.rb:44:in `require'
#0 4.128 /usr/local/bundle/gems/bundler-2.2.31/lib/bundler.rb:175:in `require'
#0 4.128 /app/config/application.rb:8:in `<main>'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
#0 4.128 /app/Rakefile:4:in `<main>'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:60:in `load'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:60:in `load'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/railties-6.1.6.1/lib/rails/commands/rake/rake_command.rb:20:in `block in perform'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/railties-6.1.6.1/lib/rails/commands/rake/rake_command.rb:18:in `perform'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/railties-6.1.6.1/lib/rails/command.rb:50:in `invoke'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/railties-6.1.6.1/lib/rails/commands.rb:18:in `<main>'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
#0 4.128 /app/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.9.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
#0 4.128 bin/rails:4:in `<main>'
#0 4.128 (See full trace by running task with --trace)
```

**Comment by denisj** [comment]
This might come from this [commit](https://github.com/mperham/sidekiq/commit/345ba002bc7035d8135d58f4c5dad6dadef298dc).

The file lib/sidekiq/job_retry.rb was requiring `require "sidekiq/scheduled"` which requires `require "sidekiq/component"`.

**Comment by mperham** [comment]
Look at the backtrace. The error is coming from sidekiq-status code loading.

**Comment by krystof-k** [comment]
You're right, thanks, I'm blind…

**Comment by denisj** [comment]
The class Sidekiq::JobRetry is including `include Sidekiq::Component` on line 66, but it is required nowhere. Do I misunderstand something?

**Comment by mperham** [comment]
> The class Sidekiq::JobRetry is including `include Sidekiq::Component` on line 66, but it is required nowhere. Do I misunderstand something?

It is only required in the Sidekiq server process. You cannot load server components in a client process.

**Comment by jensenbox** [comment]
Does this mean there is a bug in `sidekiq-status`? I am not entirely sure of the fix here.

**Comment by mperham** [comment]
Yes. They are requiring server components when they shouldn’t.

**Comment by rafaelfranca** [comment]
In my case, I'm not requiring any server component in client code. I'm trying to test a custom sidekiq logger that uses job_retry.

I think the fact that job_retry doesn't require component is a smell, that you can see in sidekiq's own test suite https://github.com/mperham/sidekiq/compare/v6.5.1...v6.5.4#diff-f9cb8a684c3b818b3e8866b7fb8a95cea48673d567a810811b531ce4c2bc6d29R4.

I see no reason why my test would need to require component other than it is requiring `job_retry`, but that dependency isn't clear anywhere. If `job_retry` need component I think it should be requiring it, or at least, a clear API for requiring `job_retry` should be provided and its dependencies.

**Comment by mperham** [comment]
That's fair enough. I can see how it's difficult to create an integration test for X if X doesn't require its dependencies.

**Comment by noctivityinc** [comment]
So for the rest of us, how do we fix this in 6.5.4?  I dont have `sidekiq-status` anywhere in my code and Im using a vanilla install of sidekiq.

**Comment by mperham** [comment]
@noctivityinc You open up your own issue with a backtrace and I diagnose it for you.

**Comment by masterkain** [comment]
@mperham will the patch https://github.com/mperham/sidekiq/commit/9d5a6d7fa7edba006db86fc520fbcd353005cf9c be included in a 6.5.x series release or 7 directly? thanks

```
+ bin/rails db:create
rails aborted!
NameError: uninitialized constant Sidekiq::Component

    include Sidekiq::Component
                   ^^^^^^^^^^^
Did you mean?  Complex
/drone/src/vendor/bundle/ruby/3.1.0/gems/sidekiq-6.5.4/lib/sidekiq/job_retry.rb:66:in `<class:JobRetry>'
/drone/src/vendor/bundle/ruby/3.1.0/gems/sidekiq-6.5.4/lib/sidekiq/job_retry.rb:61:in `<module:Sidekiq>'
/drone/src/vendor/bundle/ruby/3.1.0/gems/sidekiq-6.5.4/lib/sidekiq/job_retry.rb:6:in `<main>'
/drone/src/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
/drone/src/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
/drone/src/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
/drone/src/vendor/bundle/ruby/3.1.0/gems/sidekiq-status-2.1.3/lib/sidekiq-status/server_middleware.rb:2:in `<main>'
/drone/src/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
/drone/src/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
/drone/src/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
/drone/src/vendor/bundle/ruby/3.1.0/gems/sidekiq-status-2.1.3/lib/sidekiq-status.rb:6:in `<main>'
/drone/src/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
/drone/src/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
/drone/src/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
/drone/src/config/application.rb:20:in `<main>'
/drone/src/Rakefile:4:in `require_relative'
/drone/src/Rakefile:4:in `<main>'
/drone/src/vendor/bundle/ruby/3.1.0/gems/railties-7.0.3.1/lib/rails/commands/rake/rake_command.rb:20:in `block in perform'
/drone/src/vendor/bundle/ruby/3.1.0/gems/railties-7.0.3.1/lib/rails/commands/rake/rake_command.rb:18:in `perform'
/drone/src/vendor/bundle/ruby/3.1.0/gems/railties-7.0.3.1/lib/rails/command.rb:51:in `invoke'
/drone/src/vendor/bundle/ruby/3.1.0/gems/railties-7.0.3.1/lib/rails/commands.rb:18:in `<main>'
/drone/src/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
/drone/src/vendor/bundle/ruby/3.1.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
bin/rails:4:in `<main>'
(See full trace by running task with --trace)
```

**Comment by mperham** [comment]
That commit is on main which means it will go into 6.5.5.
